packages/iTwinUI-react/scripts/createComponent.js

```diff
@@ -175,5 +199,6 @@ inquirer
     writeFile(componentTest);
     writeFile(componentIndex);
 
-    writeFile(storiesFactory('stories/core', component));
+    writeFile(storiesFactory('apps/storybook/src', component));
+    writeFile(storyTestsFactory('apps/storybook/src', component));
   });
```

Comment:
You can use the following code here so that this script can be run from the root and from packages. It uses `process.cwd()` to get the location where the script is run from, and then adjusts the directory paths accordingly.

(couldn't get the formatting right)

Reply:
.then(({ component }) => {
    const currentDirectory = process.cwd();
    const isInPackage = currentDirectory.includes('packages');

    let directory = '';
    let storyDirectory = '';
    let levels = [];

    if (isInPackage) {
      directory = `src/core/${component}`;
      storyDirectory = '../../apps/storybook/src';
    } else {
      directory = `packages/iTwinUI-react/src/core/${component}`;
      storyDirectory = 'apps/storybook/src';
    }
    levels.push(directory);

    let componentOut = componentFactory(directory, component);
    let componentTest = componentTestFactory(directory, component);
    let componentIndex = componentIndexFactory(directory, component);
    let indexAppend = indexAppendFactory(directory, component);

    for (const level of levels) {
      makeDir(level);
    }

    appendFile(indexAppend);

    writeFile(componentOut);
    writeFile(componentTest);
    writeFile(componentIndex);

    writeFile(storiesFactory(storyDirectory, component));
    writeFile(storyTestsFactory(storyDirectory, component));
  });

Reply:
Great! Don't forget to remove the commented out code

Reply:
Thanks